modules/thermo-cycler/thermo-cycler-arduino/thermo-cycler-arduino.ino

```diff
@@ -862,8 +862,20 @@ void loop()
 #if HW_VERSION >=3
```

Comment:
If this only works in 4 shouldn't this be `HW_VERSION > 3`

Reply:
Sorry, I was kinda wrong. The V3 thermocycler lid is also able to alternately close/open because the condition for opening is that the lid should be either 'closed' or 'in_between'. Because of the 'faulty' switch position, in V3 thermocyclers, the lid will read 'in_between' when closed so it will still execute `open_cover()`

Reply:
I just moved the `#if` to  `is_front_button_pressed()` since it could be misleading here (Also, it really should have been in that function anyway)